### ISSUE TITLE
Parameter is not saved to json file for elementwise-binary-scalar-op

### ISSUE THREAD

**Issue by piiswrong** [dmlc/mxnet#995]
Parameter of _***Scalr ops are not saved to json file and gives error during loading. Reproducing:
```Python
import mxnet as mx

x = mx.sym.Variable('x')
y = 100*x
y.save('test.json')
y2 = mx.sym.load('test.json')
```
Error:
Traceback (most recent call last):
  File "test.py", line 6, in <module>
    y2 = mx.sym.load('test.json')
  File "/scratch/mxnet/python/mxnet/symbol.py", line 768, in load
    check_call(_LIB.MXSymbolCreateFromFile(c_str(fname), ctypes.byref(handle)))
  File "/scratch/mxnet/python/mxnet/base.py", line 76, in check_call
    raise MXNetError(py_str(_LIB.MXGetLastError()))
mxnet.base.MXNetError: Parameter scalar is not presented

**Comment by tqchen** [comment]
change https://github.com/dmlc/mxnet/blob/master/src/operator/elementwise_binary_scalar_op-inl.h#L255 to return param_.__DICT__();

**Comment by tqchen** [comment]
@piiswrong can you submit a fix?

**Comment by piiswrong** [comment]
I have a lot of local changes. It's going to take a while to sort them out
and push

**Comment by tqchen** [comment]
It is better to open a new branch and push changes in small PR, so they can be stagged and reviewed easily. You can always use git rebase workflow to merge the master branch back

**Comment by phecy** [comment]
I have got the same problem

**Comment by juguangliang** [comment]
@tqchen why is this problem still open, I got the same problem in the Release windows binary build 20151228, please fix it, thanks

**Comment by tqchen** [comment]
Fix PR by #1215
